app/controllers/dashboard_controller.rb

```diff
@@ -24,5 +24,32 @@ def index
     # which may not make sense when calculating total count using a date range
     @recent_donations_from_manufacturers = current_organization.donations.during(helpers.selected_range).by_source(:manufacturer)
     @top_manufacturers = current_organization.manufacturers.by_donation_count
+
+    distribution_data = helpers.received_distributed_data(helpers.selected_range)
+    @activity_chart_config = {
+      chart: {
+        type: "bar"
+      },
+      title: "",
+      xAxis: {
+        categories: distribution_data.keys,
+        title: {
+          text: nil
+        }
+      },
+      yAxis: {
+        title: {
+          text: nil
+        }
+      },
+      legend: {
+        enabled: false
+      },
+      series: [
+        {
+          data: distribution_data.values
+        }
+      ]
+    }.to_json
```

Comment:
This is a lot of lines to be in a controller... I'd either push this to the view (by passing in `@distribution_data`) or have a helper function for it.

Reply:
I think a helper function would be optimal but I haven't explored really looking at putting this in the view in this new format.

Reply:
Moved them to the view instead :)